client/src/app/components/manage-organizations/create-organization/create-organization.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, OnInit } from '@angular/core';
+import {ManageOrganizationsService} from '../../../services/manage-organizations.service';
+import {Router} from '@angular/router';
+
+@Component({
+  selector: 'app-create-organization',
+  templateUrl: './organization.component.html',
+  styleUrls: ['./organization.component.scss']
+})
+export class CreateOrganizationComponent implements OnInit {
+
+  activeStates = ['Active', 'Disabled'];
```

Comment:
This isn't being used since creation sets account status to _active_ by default
Can we remove it?